container-runtime/podman_config.sh

```diff
@@ -108,6 +108,34 @@ function selinux_is_enabled()
    return 1
 }
 
+# A small function to keep trying apt-get until other apt-get'ers finish.
+function persistent-apt-get
+{
+   eflag=0
+   if echo $- | grep -q "e"; then eflag=1; fi
+
+   MESSAGE="E: Could not get lock /var/lib/dpkg/lock-frontend."
+   APT_GET_OUTPUT=/tmp/apt-get-out-$$
+   while true; do
+      cp /dev/null $APT_GET_OUTPUT
```

Comment:
I can always guarantee, I learn something new every time I look at review.

touch $APT_GET_OUTPUT
is more commonly used.

Reply:
touch wouldn't erase the content of the file. Here the first time touch would so, but in the loop we need to make sure file is empty

Reply:
Yes, but I want to make sure the file starts out empty for each apt-get run.  Yes I could remove the file and recreate too.